lib/src/extensions/position_extension.dart

```diff
@@ -67,66 +69,142 @@ extension PositionExtension on Position {
     EditorState editorState, {
     bool upwards = true,
   }) {
-    final selection = editorState.selection;
+    final node = editorState.document.nodeAtPath(path);
+    if (node == null) {
+      return this;
+    }
+
+    final nodeRenderBox = node.renderBox;
+    if (nodeRenderBox == null) {
+      return this;
+    }
+
+    final nodeSelectable = node.selectable;
+    if (nodeSelectable == null) {
+      return this;
+    }
```

Comment:
We can consolidate these three if-statements
```suggestion
    final node = editorState.document.nodeAtPath(path);
    final nodeRenderBox = node?.renderBox;
    final nodeSelectable = node?.selectable;
    if (node == null || nodeRenderBox == null || nodeSelectable == null) {
      return this;
    }
```